CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ This log was last generated on Wed, 31 May 2023 21:24:48 GMT and should not be m
 
 <!-- Start content -->
 
+## 1.0.1 (2023-06-27)
```

Comment:
the changelog would get auto-generated/updated by the changelog file you created.
everything you mentioned here should be included in the chnage/*.json files